app/views/RoomMembersView/index.js

```diff
@@ -389,16 +391,31 @@ class RoomMembersView extends React.Component {
 
 	fetchMembers = async() => {
 		const {
-			rid, members, isLoading, allUsers, end
+			rid, members, isLoading, allUsers, end, room, filtering
 		} = this.state;
+		const { serverVersion } = this.props;
+		const { t } = room;
+		let newMembers;
 		if (isLoading || end) {
 			return;
 		}
 
 		this.setState({ isLoading: true });
 		try {
-			const membersResult = await RocketChat.getRoomMembers(rid, allUsers, members.length, PAGE_SIZE);
-			const newMembers = membersResult.records;
+			if (compareServerVersion(serverVersion, '3.16.0', methods.greaterThanOrEqualTo)) {
+				const membersResult = await RocketChat.getMembers({
```

Comment:
@gerzonc how about reusing the same function name and we just check the server version?
We would have `getRoomMembers` for both.